solutions/client-landing-zone/client-folder/standard/applications-infrastructure/host-project/network/public-dns.yaml

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+# Public Client DNS subzone
+# SC-22
+apiVersion: dns.cnrm.cloud.google.com/v1beta1
+kind: DNSManagedZone
+metadata:
+  name: client-name-standard-public-dns # kpt-set: ${client-name}-standard-public-dns
+  namespace: client-name-networking # kpt-set: ${client-name}-networking
+  annotations:
+    cnrm.cloud.google.com/project-id: host-project-id # kpt-set: ${host-project-id}
+    config.kubernetes.io/depends-on: resourcemanager.cnrm.cloud.google.com/namespaces/ssc-spc-projects/Project/host-project-id # kpt-set: resourcemanager.cnrm.cloud.google.com/namespaces/ssc-spc-projects/Project/${host-project-id}
```

Comment:
`ssc-spc` should be `client-name`

Reply:
Why would we hardcode `ssc-spc` as the `client-name`? Aren't clients getting their own DNS subzone?

Reply:
What about the core DNS? Is SSC responsible for hosting all DNS resolutions? Anyways, not having a `client-name` variable makes the code less reusable for outside parties, no?

Reply:
I was looking at the wrong place. Good catch. Fixed. Please resolved if it looks good now. Thanks!